### PR TITLE
Feature Enhancement to px-icon

### PR DESCRIPTION
**Pull request by mallik-svmx**
Feature Enhancement:
- Provides a configurable option to change the icon stroke-width. Default `strokeWidth` is 1  E.g.
stroke-width

- Supports Hebrew language RTL Support and fixes px-nav icon spacing issue

### CODE REVIEW

**Comment by mallik-svmx** [comment]
@joshsylvester @sheela-svmx @arongraham-ge - please review. Thanks!

**Comment by mallik-svmx** [comment]
@benjaminliugang - Can you please sign the CLA? Thanks!

File: HISTORY.md [hunk 1]
```diff
@@ -1,4 +1,29 @@
 v3.1.0
+==================
```
Comment (evanjd) [inline HISTORY.md:2]: The ordering is all wrong here.

It goes from 3.1.0, back to 2.2.0, back to 3.1.0 again.

**Review by evanjd — CHANGES_REQUESTED** [review]
Can you please remove the package-lock.json changes - no packages were changed so the lock file shouldn't be touched.

Also, this PR needs to be rebased to fix the author on commit [9797829c2](https://github.com/predixdesignsystem/px-app-nav/pull/112/commits/9797829c2b1801cb5c9b36d34d098ab42e698622). The commit was authored with e-mail benjamin.liu@ge.com, which doesn't match @benjaminliugang primary github e-mail of liugangyanxing@163.com.

**Pull request closed without merging**